### ISSUE TITLE
Writer fails on chown

### ISSUE THREAD

**Issue by silkentrance** [npm/fstream#36]
In lib/writer.js it reads

```
195 function endChown (me, want, current, path, cb) {
196   // Don't even try it unless root.  Too easy to EPERM.
197   if (process.platform === "win32") return cb()
198   if (!process.getuid || !process.getuid() == 0) return cb()
```

Here, the expression

```
198   if (!process.getuid || !process.getuid() == 0) return cb()
```

fails due to `!process.getuid() === 0` being the wrong statement, it should read
`(process.getuid() !== 0)` instead.

**Comment by silkentrance** [comment]
See also https://github.com/npm/npm/issues/7120#issuecomment-69822522

**Comment by nathan7** [comment]
Still pending an npm publish from @othiym23 or @isaacs for this to be actually fixed.

**Comment by silkentrance** [comment]
npm includes fstream in a rather old version and does not depend on it package wise.
It should be updated there, too.

**Comment by silkentrance** [comment]
@isaacs @othiym23 how about making this available to the public...

**Comment by othiym23** [comment]
This is published as part of `fstream@1.0.4` and I'm currently testing `fstream@1.0.4` in npm, so it will probably be there shortly.

**Comment by silkentrance** [comment]
@othiym23 Great, thanks.
